sanic/request.py

```diff
@@ -106,6 +106,15 @@ def __bool__(self):
             return True
         return False
 
+    def body_init(self):
+        self.body = []
+
+    def body_push(self, data):
```

Comment:
Yeah, I'm really open to suggestions regarding that matter and, instead of `init_body`, why not `start_body`?

Reply:
@vltr I like that too. If we go that route, then these might look mode readable `start_body`, `update_body`, `finalize_body`? What you do think?

Reply:
Sounds good to me.

I would like to welcome any other inputs as well ... @ahopkins @abuckenheimer @sjsadowski ?

Reply:
They seem a little more verbose, and there is a sort of namespacing consistency with starting them with `body_` so that it is obvious that they are working together.

If it were a group of methods that were intentionally being exposed to the end user developers, then perhaps something more semantic would make sense for API ease of use. But since they are going to be one layer down (meant more for "power user" consumption), I prefer grouping them in an obvious pattern.

Reply:
This makes sense too. I'll vote for let it be as it's now since this PR had been through a lot so it can be properly merged into master :sweat_smile:

Reply:
@vltr @ahopkins 👍